recipes/qhube-dask/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set name = "qhub-dask-meta" %}
+{% set version = "2021.03.04" %}
+
+package: 
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+  run:
+    - dask =2.14.0
+    - distributed =2.14
+    - dask-gateway =0.6.1
```

Comment:
These pins are really restrictive. Are they actually needed?

Reply:
I think so...

Reply:
Not my package so if you are happy with it, then I am too.